extensions/src/main/java/org/apache/falcon/extensions/store/ExtensionStore.java

```diff
@@ -269,6 +269,18 @@ public String registerExtension(final String extensionName, final String path, f
         throws URISyntaxException, FalconException {
         Configuration conf = new Configuration();
         URI uri = new URI(path);
+        if (uri.getScheme() == null){
```

Comment:
Nit : May be club all 3 checks. And say, "Extension path must contain a scheme, authority and path. Example, hdfs://<namenode>:port/projects/falcon/extension"